src/main/webapp/scripts/map.js

```diff
@@ -55,26 +55,33 @@ window.onload = function() {
   initMap();
 }
 
+let map;
+
 /* Set Up Map */
 function initMap() { 
   /* 'Hard Coded' origin, Google UK Pancras Square */
   const origin = {
     lat: 51.533364, 
     lng: -0.125777
   }
-  const map = new google.maps.Map(document.getElementById("map"), {
+  
+  map = new google.maps.Map(document.getElementById("map"), {
     center: origin,
     zoom: 15
   });
-  map.addListener("click", e => {
-    placeMarkerAndPanTo(e.latLng, map);
-  });
+  
+  /* Get all stored markers */
+  fetchMarkers();
```

Comment:
It feels like we need some explanation here. We are fetching markers from backend but placing hardcoded markers here.

Reply:
The transition from hardcoded to just backend (fetched from the Datastore) is next on the list, thanks!